docs/learn/learn-account-abstraction.md

```diff
@@ -0,0 +1,64 @@
+---
+id: learn-account-abstraction
+title: Polkadot's Account Abstraction
+sidebar_label: Account Abstraction
+description: Polkadot's Native Account Abstraction.
+keywords: [account, polkadot account, account abstraction, proxy, multisig, batch]
+slug: ../learn-account-abstraction
+---
+
+## Your Keys, Your Responsibility
+
+Account abstraction addresses the challenges associated with managing cryptographic keys that on
+blockchains represent accounts and, more generally, your digital identity. In Web3, you digitally
+sign any transaction or, more generally, any message using your private key. Data are recorded on a
+public ledger (blockchain) whose multiple copies of it are stored in computers participating in a
+P2P network. Assuming the network is sufficiently decentralized (meaning that those computers do not
+belong to a few operators), those computers do not own your data, you do since you can verify you
+signed them with your key.
+
+While they grant users control and ownership, losing keys results in a loss of digital assets and a
+loss and fragmentation of your digital identity since you will need to create a new account with a
+new key after losing a key. This poses a hurdle for both users and developers regarding security and
+adoption.
+
+## Definition of Account Abstraction
+
+Account abstraction introduces a layer of on-chain logic that controls an account, typically in the
+form of a smart contract. This abstracts some key management complexities, reducing the likelihood
+of compromise. It allows for dynamic changes in an account’s operation without altering the core
+protocol.
```

Comment:
How does it reduce the likelihood of a compromise? What are the dynamic changes? What core protocols need to be altered if Account abstraction is not present?

Reply:
This is partially answered in the next paragraph, but it reduces the importance of having to manage the private key as the _sole_ method of access for a user's on-chain representation.  Of course, in our context, this depends on things like the social recovery pallet.

Core protocols in the Substrate context include either: 

1. Have the ability to do this with ink! (to my knowledge possible, but no one has done it)
2. A pallet to do this (social recovery)

In short, either the **runtime** needs to be altered or the implementation/enforcement of an ink! smart contract.